### PR TITLE
[ZEPPELIN-834]Handle Note-name with spaces

### PR DESCRIPTION
**Pull request by ravicodder**
### What is this PR for?
Handle notename to have only white spaces. 

### What type of PR is it?
[Bug Fix ]

### What is the Jira issue?
[ZEPPELIN-834] (https://issues.apache.org/jira/browse/ZEPPELIN-834)

### How should this be tested?
Create note with name having only white spaces.
you should be able to see note id for those note.

### Screenshots (if appropriate)
Before:
screen shot 2016-04-29 at 10 40 27 am
screen shot 2016-04-29 at 10 37 56 am
screen shot 2016-04-29 at 10 36 23 am
 After:
screen shot 2016-04-29 at 10 40 17 am
screen shot 2016-04-29 at 10 39 57 am
screen shot 2016-04-29 at 10 38 16 am

### Questions:
* Does the licenses files need update?NO
* Is there breaking changes for older versions?NO
* Does this needs documentation?NO

### CODE REVIEW

File: docs/manual/notebookashomepage.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ you need to do is use our %angular support.
           <i style="font-size: 15px;" class="icon-notebook"></i> Create new note</a></h5>
           <ul style="list-style-type: none;">
             <li ng-repeat="note in home.notes.list track by $index"><i style="font-size: 10px;" class="icon-doc"></i>
-              <a style="text-decoration: none;" href="#/notebook/{{note.id}}">{{note.name || 'Note ' + note.id}}</a>
+              <a style="text-decoration: none;" href="#/notebook/{{note.id}}">{{note.name.trim()==='' && 'Note ' + note.id || note.name}}</a>
```
Comment (prabhjyotsingh) [inline docs/manual/notebookashomepage.md]: can you move this logic to a function

**Comment by prabhjyotsingh** [comment]
I think you should refactor all of these `{{note.name.trim()==='' && 'Note ' + note.id || note.name}}` to a common function.

**Comment by ravicodder** [comment]
@prabhjyotsingh thanks for review.
Refactored all the logic into common function.

**Comment by prabhjyotsingh** [comment]
LGTM!

**Comment by prabhjyotsingh** [comment]
Merging is no more discussion.

**Pull request closed without merging**